PMM-10286 Misleading message (#843)
percona/pmm-doc: docs/using/alerting.md

# Integrated Alerting

!!! caution alert alert-warning "Disclaimer"

>     Integrated Alerting is currently in [Technical Preview](../details/glossary.md#technical-preview). This means that this feature is still under development and is subject to change. For other cases, see the [Alternative alerting options](#alternative-alerting-options) section.

Reviewer: ```suggestion
    Integrated Alerting is currently in [Technical Preview](../details/glossary.md#technical-preview). This means that this feature is still under development and is subject to change. For other alerting options, see [Alternative alerting options](#alternative-alerting-options).
```

>     Integrated Alerting is currently in [Technical Preview](../details/glossary.md#technical-preview). This means that this feature is still under development and is subject to change. For other cases, please see  [Alternative alerting options](#alternative-alerting-options) section

Reviewer 2: ```suggestion
    Integrated Alerting is currently in [Technical Preview](../details/glossary.md#technical-preview). This means that this feature is still under development and is subject to change. For other cases, see the [Alternative alerting options](#alternative-alerting-options) section.
```

To reactivate silenced alerts, click **Unsilence All** or click on the disabled bell icon <i class="uil uil-bell-slash"></i> corresponding to the alerts you want to reactivate.

## Alternative alerting options

> | Use | <i class="uil uil-thumbs-up"></i> **Benefits** | <i class="uil uil-thumbs-down"></i> **Drawbacks**

Reviewer: ```suggestion
| Use | <i class="uil uil-thumbs-up"></i> **Pros** | <i class="uil uil-thumbs-down"></i> **Cons**
```

|---|---|---

> | [External Alertmanager](/how-to/configure.html#alertmanager-integration) | 1. A way to integrate PMM when a company has an existing [Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/). | 1.  For experts who use it already 

Reviewer: ```suggestion
| [External Alertmanager](/how-to/configure.html#alertmanager-integration) | A way to integrate PMM when an organization has an existing [Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/). | For experts who already use it
```

> | **Recommended**<br>[Native Grafana Alerting](https://grafana.com/docs/grafana/v8.5/alerting/) | 1. Build-in<br>2. Feature-rich. | 1. Available in the latest PMM versions only.

Reviewer: ```suggestion
| [Native Grafana Alerting](https://grafana.com/docs/grafana/v8.5/alerting/) | 1. Built-in<br>2. Feature-rich. | Available in the latest PMM versions only.
```

Reviewer: Is it a good idea to recommend Native Grafana Alerting as we want to promote Integrated Alerting...

Author: no,   in 2.31 we;ll merge  Integrated Alerting and Native Grafana alerting and  -  Native Grafana alerting  will be the base for this feature

> | [Integrated Alerting](#) | 1. Simple. | 1. Technical Preview.<br>2. Less integrations then in _Native Grafana Alerting_.

Reviewer: ```suggestion
| [Integrated Alerting](#) | Simple. | 1. Technical Preview.<br>2. Less integrations as compared to _Native Grafana Alerting_.
```

> | [Legacy Grafana Alerting](https://grafana.com/docs/grafana-cloud/legacy-alerting/) | N/A | 1. Deprecated. 

Reviewer: If it's deprecated, why are we mentioning it here...
